packages/spector/src/cli/cli.ts

```diff
@@ -1,4 +1,5 @@
-import { join, resolve } from "path";
+import * as fs from "fs";
```

Comment:
import from `fs/promises` and try to no do `import * as fs`

Reply:
Done